examples/js/lights/LightProbeGenerator.js

```diff
@@ -116,6 +116,105 @@ THREE.LightProbeGenerator = {
 
 		return new THREE.LightProbe( sh );
 
+	},
+
+	fromCubeRenderTarget: function ( renderer, renderTarget ) {
```

Comment:
Can you explain what is wrong for you?

Reply:
I think color correction in threejs is a little bit messy.
So i should simply assume that the renderTarget is linear. What do you think?
Color correction should be applied as a render pass if the user wants it.
It would be more simple to understand and all buffer would be in linear color space has it should be without wondering in which color space we are working on!

Reply:
I think you can follow the same pattern in your method I used in #17385 and decode to linear based on the encoding of the `WebGLRenderTargetCube`.

Reply:
I think the function signature should be
```js
fromRenderTargetCube: function ( renderer, renderTargetCube ) {
```